.github/README.md

```diff
@@ -60,26 +82,39 @@ After that, you can just open the `UndertaleModTool.sln` file in Visual Studio,
 Alternatively, you can also compile via command line, like so: `dotnet publish UndertaleModTool --no-self-contained -p:PublishSingleFile=true -c release -r win-x86`
 You can adjust win-x86 to other RIDs, such as win-x64 or win-arm.
 
```

Comment:
Technically not part of this PR, but the compilation instructions should get rewritten. I may do it at one point later, but the key aspects are:
- Windows is only needed for compiling the GUI
- due to all projects now having rolling-forward enabled, .NET core 5 is not needed anymore, rather anything above it
- Visual Studio is not needed; only the SDK. Visual studio just invokes the command line parameters, which can be done by anything (visual studio code, rider, vim, etc.)
- the Cli parameters are technically not necessary
- no description of what RIDs are.

Reply:
This wasn't addressed in any way, why was this closed?

Reply:
Rewrote the part.
```md
In order to compile the repo yourself, the `.NET Core 5 SDK` or later is required.

The following projects can be compiled:  
- `UndertaleModLib`: The core library used by all other projects.
- `UndertaleModCli`: A command line interface for interacting with GameMaker data files and applying scripts. Currently is very primitive in what it can do.
- `UndertaleModTool`: The main graphical user interface for interacting with GameMaker data files. **Windows is required in order to compile this**.

#### Compiling Via IDE
- Open the `UndertaleModTool.sln` in the IDE of your choice (Visual Studio, JetBrains Rider, Visual Studio Code etc.)
- Select the project you want to compile
- Compile

#### Compiling Via Command Line
- Open a terminal and navigate to the directory of `UndertaleModTool.sln`
- Execute `dotnet publish <Project>` where `<Project>` is one of the projects listed above.
You can also provide arguments for compiling, such as `--no-self-contained` or `-c release`. For a full list of arguments, consult the [Microsoft Documentation](https://docs.microsoft.com/dotnet/core/tools/dotnet-publish)
```